### PR TITLE
Help Wanted: Secretary [DNM]

### PR DESCRIPTION
**Pull request by cornercube**
## About The Pull Request

New Middle Management position for Oasis: The Secretary
helps manage diplomacy with other factions, listens to concerned citizens, has affairs with the mayor, 
starts with loads of paper, a pip boy, and a silenced pistol 

to do:
cannot solve linter problem
need mapping to actually add the spawn point
## Why It's Good For The Game

a fluff role for the town
on high-pop, the sheriff and mayor are going to overworked, so this provides a release valve for trivial things
also gives a training position for people who want to eventually take mayor later on, without being plunged neck deep

## Pre-Merge Checklist
- [ ] Your Pull Request contains no breaking changes
- [ ] You tested your changes locally, and they work.
- [x] There are no new Runtimes that appear.
- [x] You documented all of your changes.

## Changelog
:cl:
add: new role: the secretary
add: a special briefcase holding their stuff
del: the secretary loadout for the citizen

/:cl:

### CODE REVIEW

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -107,6 +107,7 @@
 #define F13PROSPECTOR	(1<<4)
 #define F13DEPUTY		(1<<5)
 #define F13MAYOR		(1<<6)
+#define F13SECRETARY	(1<<6)
```
Comment (Tron-117) [inline code/__DEFINES/jobs.dm:110]: Not exactly sure what you'd call it, but you see the number thing next to the role? Yeah, that's wrong. It's supposed to be 7. And even after you changed that, you have to update the numbers (basically increasing them by 1) for the rest of the Oasis roles after the Secretary role to fit in now.

Comment (cornercube) [reply code/__DEFINES/jobs.dm:110]: alright
was just following the den doctors update as a template which did the same thing
fixing it now

Comment (cornercube) [reply code/__DEFINES/jobs.dm]: didnt do anything, linters still stuck

Comment (Tron-117) [reply code/__DEFINES/jobs.dm]: Hm, alright. On lines 94, 98, and 120, there are unnecessary commas which ends lists. I thiiiiink that breaks stuff. Try removing those and see if it works.

Comment (cornercube) [reply code/__DEFINES/jobs.dm]: still nothing

also, what does this mean?
its the only error i can see in the logs?
Run yogstation13/DreamAnnotate@v1
Error: ENOENT: no such file or directory, open 'output-annotations.txt'

Comment (Tron-117) [reply code/__DEFINES/jobs.dm]: Okay, after digging through some PRs and their checks, it seems that you didn't indent properly somewhere in code/modules/jobs/job_types/oasis.dm. Not sure where yet, but I'm looking.

Comment (cornercube) [reply code/__DEFINES/jobs.dm]: yeah, accidentally made it worse trying to look for the bad indent

Comment (cornercube) [reply code/__DEFINES/jobs.dm]: gonna try and re-do this from scratch

**Pull request closed without merging**